transit_odp/browse/common.py

```diff
@@ -73,6 +77,9 @@ def get_service_traveline_regions(ui_ltas):
     return "|".join(
         [
             str(TravelineRegions(admin_area.traveline_region_id).label)
-            for admin_area in AdminArea.objects.filter(ui_lta__in=ui_ltas).all()
+            for admin_area in AdminArea.objects.filter(ui_lta__in=ui_ltas)
+            .order_by("traveline_region_id")
```

Comment:
we can apply order_by after distinct so less records need to be operated on

Reply:
Updated